TestCases/aeroelastic/aeroelastic_NACA64A010.cfg

```diff
@@ -254,8 +254,11 @@ DEFORM_TOL_FACTOR = 0.001
 %                                          WALL_DISTANCE, CONSTANT_STIFFNESS)
 DEFORM_STIFFNESS_TYPE= INVERSE_VOLUME
 %
-% Visualize the deformation (NO, YES)
-VISUALIZE_DEFORMATION= NO
+% Visualize the surface deformation (NO, YES)
+VISUALIZE_SURFACE_DEF= NO
```

Comment:
Do you think it is cheap enough to turn the surface file writing by default? I think at least the surface visualization can be very important...

Reply:
Agree! Done!